app/models/concerns/publishable.rb

```diff
@@ -24,4 +27,14 @@ def publish_content_item
   def publishing_api
     @publishing_api ||= PolicyPublisher.services(:publishing_api)
   end
+
+  def publish_rummager_artefact!
```

Comment:
Don't feel this should block this pull request (it can be refactored later), but I think we could better encapsulate the act of indexing something in rummager by having a `PolicyIndexer` that does the work of this method instead of the presenter, e.g. `PolicyIndexer.new(policy).index!`. That way, the model doesn't need to know any details about rummager, or the services, or how it needs to be presented. Same goes for publishing to publishing API.